presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaColumnHandle.java

```diff
@@ -167,7 +167,8 @@ public boolean isInternal()
 
     ColumnMetadata getColumnMetadata()
     {
-        return new ColumnMetadata(name, type, null, hidden);
+        return ColumnMetadata.builder(name, type)
+                .setHidden(hidden).build();
```

Comment:
tiny nit: when we do the setXX on different lines, we normally put .build() on its own line too.